### PR TITLE
 Raylife AP Automated Tests - LPS-182947 | master

### PR DESCRIPTION
**Pull request by itstali**
https://issues.liferay.com/browse/LPS-182947

### CODE REVIEW

**Comment by itstali** [comment]
ci:test:sf

File: modules/apps/site-initializer/site-initializer-raylife-ap/site-initializer-raylife-ap-test/src/testFunctional/tests/RaylifeAPIncompleteApplication.testcase [hunk 1]
```diff
@@ -0,0 +1,254 @@
...
+
+			for (var row : table ${rawTable}) {
+				var label = "${row[0]}";
+				var value = "${row[1]}";
+
+				AssertElementPresent(
+					key_text = ${label},
+					key_value = ${value},
+					locator1 = "RaylifeApplicationModal#DETAIL_INFO");
+			}
```
Comment (luanakarla) [inline modules/apps/site-initializer/site-initializer-raylife-ap/site-initializer-raylife-ap-test/src/testFunctional/tests/RaylifeAPIncompleteApplication.testcase:134]: This is very good. Great job!

**Review by luanakarla — APPROVED** [review]
Hey @itstali, thanks for working on it!

The code looks good to me!

I'm forwarding it.

**Comment by luanakarla** [comment]
ci:forward

**Comment by luanakarla** [comment]
ci:test:sf

**Comment by luanakarla** [comment]
ci:forward

**Pull request closed without merging**